src/LandingPage/FAQ.tsx

```diff
@@ -14,26 +15,26 @@ interface qnaEntry {
 }
 const qnaEntries: qnaEntry[] = [
   {
```

Comment:
@JLamperti : I've reordered the entries, with the idea that the FAQ is primarily user-facing. Let me know if that's fine with you! :)